be/src/storage/tablet_meta.h

```diff
@@ -158,6 +158,7 @@ class TabletMeta {
     Version max_version() const;
 
     inline TabletState tablet_state() const;
+    // NOTE: Normally you should NOT call this method directly but call Tablet::set_tablet_state().
```

Comment:
what does the annotation means?

Reply:
This is a dangerous method, it may change the state from SHUTDOWN to RUNNING again, which should not happen in normal cases.